src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Platform/AzureManagedServiceAuthenticator.cs

```diff
@@ -21,17 +22,30 @@ public AzureManagedServiceAuthenticator(bool useAzureManagedIdentity, string azu
             _useAzureManagedIdentity = useAzureManagedIdentity;
             _azureServiceTokenProviderResource = azureServiceTokenProviderResource;
             _tenantId = tenantId;
-            _azureServiceTokenProvider = new AzureServiceTokenProvider();
+            _defaultAzureCredential = new DefaultAzureCredential(new DefaultAzureCredentialOptions
+            {
+                ExcludeEnvironmentCredential = true,
```

Comment:
Doesn't this prevent useful local debugging/testing possibilities (e.g. when debugging a program using this sink in VS or running a sink from an Az authenticated powershell instance)?

Reply:
This change is rollbacked